auto_install/install.sh

```diff
@@ -444,43 +480,79 @@ preconfigurePackages(){
 	# Install packages used by this installation script
 	# If apt is older than 1.5 we need to install an additional package to add
 	# support for https repositories that will be used later on
-	if [[ -f /etc/apt/sources.list ]]; then
+	if [ "${PKG_MANAGER}" == 'apt-get' ] && [ -f /etc/apt/sources.list ]
+	then
 		INSTALLED_APT="$(apt-cache policy apt | grep -m1 'Installed: ' | grep -v '(none)' | awk '{print $2}')"
 		if dpkg --compare-versions "$INSTALLED_APT" lt 1.5; then
 			BASE_DEPS+=("apt-transport-https")
 		fi
 	fi
 
 	# We set static IP only on Raspberry Pi OS
-	if checkStaticIpSupported; then
+	if [ "${PKG_MANAGER}" == 'apt-get' ] && checkStaticIpSupported
+	then
 		BASE_DEPS+=(dhcpcd5)
+	elif [ "${PKG_MANAGER}" == 'apk' ] && checkStaticIpSupported
+	then
+		BASE_DEPS_ALPINE+=(dhcpcd)
+	fi
```

Comment:
This part can be reverted. `dhcpcd` is only aimed to be configured on Raspberry Pi OS, since it ships with `dhcpcd` is default network service. Also revert the change in `checkStaticIpSupported` accordingly, so it returns false on Alpine, which doesn't use `dhcpcd` by default, does it?

Reply:
Alpine use BusyBox, so have the BusyBox DHCP tool (`busybox udhcpc`)

Reply:
Then installing `dhcpcd` would likely break it, at best is unnecessary only. So that should be removed, leaving the static IP configuration up to the admin.

Reply:
Already done